### DISCUSSION TITLE
Things have certainly changed.. Github..! I can't even log into the old site..

### DISCUSSION THREAD

**Discussion by zmixnyc** [RomanKubiak/ctrlr#401] [General]
Hello everybody..

Things have certainly changed.. Github..! 

I can read the ctrlr.org site but can not  log in, it's just a cyclic login page.

Not even sure where to begin here...!

I'll have a look.

**Comment by Godlike-Productions** [comment]
Yeah, the ctrlr site was hit by spammers and I presume wordpress security holes.  It's been read only for ages.  Most stuff is happening here nowdays.

**Comment by dobo365** [comment]
Indeed. This whole stuff is just a mess...
The old site was better for panel display, forum exchange, visibility of releases
The new site is better for development discussions, code maintenance
I have still reached the old site end november but since then, indeed, impossible to log in.

This situation could be better if some of us had the admin rights and could clean/watch things.
If WP was so bad, it would not been used by people. So I guess it is a question of settings or management (my guess).
Or may be it is the WP forum that is bad and WP web site development that is OK...

**Reply by damiensellier** [reply]
Wordpress is rock solid. I use it all the time for websites. The problem is the lack of updates and the spam filters/writing credentials.

**Reply by Godlike-Productions** [reply]
Wordpress is ok as blogging software, if you stay on top of outdated and unsupported extensions and keep security updates up to date.

My experience has been that it's piles and piles of wonky extensions, to do anything beyond its original functionality.  It's security relies on all the extensions doing security properly.

For any substantial site a proper CMS like typo3 is a much better undertaking.  Secure from the core, and designed to build a website. Wordpress hurts my brain, whenever I'm forced to use it.
